pkg/resources/task_acceptance_test.go

```diff
@@ -27,10 +28,6 @@ import (
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-// TODO(SNOW-1348116 - next pr): More tests for complicated DAGs
```

Comment:
not sure if the added test is enough for complicated DAG, let's discuss tomorrow

Reply:
Added TODO at the top of the test file